backend/voting/models.py

```diff
@@ -7,10 +7,11 @@
 
 class Vote(TimeStampedModel):
     VALUES = Choices(
-        (0, "not_interested", _("Not Interested")),
-        (1, "maybe", _("Maybe")),
-        (2, "want_to_see", _("Want to See")),
-        (3, "must_see", _("Must See")),
+        (1, "not_interested", _("Not Interested")),
+        (2, "maybe", _("Maybe")),
+        (3, "want_to_see", _("Want to See")),
+        (4, "must_see", _("Must See")),
+        (5, "love_it", _("Love it")),
```

Comment:
let's find a better name for this :P @patrick91 or maybe just use Integers instead of ChoiceField? we do not use the text anywhere...